### PR TITLE
Prepare for 1.2.4 release "Kalalau trail"

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
This PR will change the default memory requirements for the false discovery estimation and output visualization.

It therefore fixes the problems encountered when running it on the AWS system. 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [x] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/diaproteomics/tree/master/.github/CONTRIBUTING.md)
  - [x] If necessary, also make a PR on the nf-core/diaproteomics _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v.1.2.4 - [27.04.21]
+
+- Raise memory requirements for FDR estimation
```
Comment (Leon-Bichmann) [inline CHANGELOG.md]:
```suggestion
- Raise memory requirements for FDR estimation and output visualization
```

**Review by ggabernet — APPROVED** [review]
Looks good to me! Let's hope the AWS tests run through now!

**Review by ewels — APPROVED** [review]
LGTM. Maybe worth triggering the tests manually from `master` once merged? Then we can confirm that they work before making a release..

**Comment by Leon-Bichmann** [comment]
alright, will do that. On the dev branch it passed already, so I think it will be fine.

**Comment by ggabernet** [comment]
> LGTM. Maybe worth triggering the tests manually from `master` once merged? Then we can confirm that they work before making a release..

This should not be done as then the directory already exists, and the release tests will break. Additionally, the release container is not yet built so the tests will fail. Testing them on the `dev` branch before merging to `master` should be fine though.

**Comment by ggabernet** [comment]
After removing the already created directory and triggering the tests manually they are running now

**Pull request merged**